buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -1,7 +1,7 @@
 object Versions {
     val marathon = System.getenv("DEPLOY_VERSION_OVERRIDE") ?: "0.7.0"
 
-    val kotlin = "1.4.20"
+    val kotlin = "1.4.31"
```

Comment:
1.4.32 - latest stable

Reply:
I checked release notes of Gradle 7.0, they upgraded Kotlin to 1.4.31(
[#16330] - Upgrade embedded Kotlin to 1.4.31). I hope upgrade to 1.4.32 will not cause any issue, but I would prefer to keep version same. Let's check it later